vhdbuilder/packer/install-dependencies.sh

```diff
@@ -117,8 +117,21 @@ else
   cliTool="docker"
 fi
 
-RUNC_VERSION=$(runc --version | head -n1 | sed 's/runc version //')
-echo "  - runc version ${RUNC_VERSION}" >> ${VHD_LOGS_FILEPATH}
+INSTALLED_RUNC_VERSION=$(runc --version | head -n1 | sed 's/runc version //')
+echo "  - runc version ${INSTALLED_RUNC_VERSION}" >> ${VHD_LOGS_FILEPATH}
+
+## for ubuntu-based images, cache multiple versions of runc
+if [[ $OS == $UBUNTU_OS_NAME ]]; then
+  RUNC_VERSIONS="
+  1.0.0-rc92
+  1.0.0-rc95
+  "
+  for RUNC_VERSION in $RUNC_VERSIONS; do
+    downloadDebPkgToFile "moby-runc" ${RUNC_VERSION} ${RUNC_DOWNLOADS_DIR}
+    echo "  - [cached] runc ${RUNC_VERSION}" >> ${VHD_LOGS_FILEPATH}
+  done
+fi
```

Comment:
should this be in ensureRunc() so all the runc install/caching logic stay together?

Reply:
also rc95 is used for docker nodes as well, did Brian confirm it's ok?

Reply:
ensureRunc is for installing a specific runc version though - this is to just cache runc binaries during vhd build. i think they should be separate. 

checking w/ brian

Reply:
docker w/ r95 is good and recommended